### ISSUE TITLE
Boxplot on roadmap?

### ISSUE THREAD

**Issue by michi-zuri** [terezka/elm-charts#108]
When describing medical trials, boxplots are used all over the place, they look like this:
https://discourse.elm-lang.org/t/looking-for-all-the-charts/7137/18

Wikipedia has [a nice article about boxplots](https://en.wikipedia.org/wiki/Box_plot) with good examples.

**Comment by terezka** [comment]
Yeah, actually thought this would be fun to add, but haven't gotten around to it! You can probably make it with the internal api [`Chart.Svg`](https://package.elm-lang.org/packages/terezka/elm-charts/latest/Chart-Svg) and [`Chart.custom`](https://package.elm-lang.org/packages/terezka/elm-charts/latest/Chart#custom). See example of [custom charts](https://www.elm-charts.org/documentation/navigation/custom-chart-elements).
